test/Microsoft.AspNet.Mvc.Testing.Test/ControllerTestHelperTest.cs

```diff
@@ -0,0 +1,198 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
It's **really** hard to get a grasp on whether or not this design stands up without some compelling tests and controller code to look at.

Examples:
https://github.com/aspnet/MusicStore/blob/master/src/MusicStore/Controllers/AccountController.cs#L227
https://github.com/aspnet/MusicStore/blob/master/src/MusicStore.Spa/Apis/AlbumsApiController.cs#L144